### PR TITLE
Implement backup codes

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
In relation to https://github.com/seamapi/ruby/issues/58

### CODE REVIEW

File: lib/seam/clients/access_codes.rb [hunk 1]
```diff
@@ -3,7 +3,9 @@
 module Seam
   module Clients
     class AccessCodes < BaseClient
-      def get(access_code_id)
+      def get(access_code_or_id)
```
Comment (seveibar) [inline lib/seam/clients/access_codes.rb]: I dont think we're going to roll out this feature, i know python does it but i think it creates too much inconsistency/variations of function calls

**Review by seveibar — DISMISSED** [review]

File: lib/seam/clients/access_codes.rb (hunk 1 above)
Comment (seveibar) [reply lib/seam/clients/access_codes.rb]: Can we remove it from this pr?

Comment (andrii-balitskyi) [reply lib/seam/clients/access_codes.rb]: @seveibar of course!

Comment (andrii-balitskyi) [reply lib/seam/clients/access_codes.rb]: https://github.com/seamapi/ruby/pull/64/commits/65b83d2c2fb7d33b84a1ecb9c9303d632a2fbd6c

**Review by seveibar — DISMISSED** [review]

**Review by seveibar — APPROVED** [review]

**Pull request merged**